importer/Import/Repodata.hs

```diff
@@ -94,7 +94,11 @@ loadRepoFromURI uri = do
 
  where
     appendOrThrow :: T.Text -> URI
-    appendOrThrow path = appendURI uri (T.unpack path) `throwIfNothing` RepoException
+    appendOrThrow path = let
+          -- If the URI path already ends in a slash, don't add an extra one
+          new_path = if last (uriPath uri) == '/' then T.unpack path else "/" ++ T.unpack path
```

Comment:
thanks, will update but why is not this change not enough or am I missing something